backend/application/syson-sysml-import/src/main/java/org/eclipse/syson/sysml/ASTTransformer.java

```diff
@@ -91,6 +93,54 @@ public Resource convertResource(final InputStream input, final ResourceSet resou
     private void postResolvingFixingPhase(List<EObject> rootSysmlObjects) {
         for (EObject root : rootSysmlObjects) {
             this.fixTransitionUsageImplicitSource(root);
+            this.fixOperatorExpressionUsedAsRanges(root);
+        }
+    }
+
+    private void fixOperatorExpressionUsedAsRanges(EObject root) {
+        // Only get the OperatorExpressions used in MultiplicityRange. Based on KerML 8.2.5.8.1 OperatorExpression
+        // referring to the ".." function can exist, but the specification does not allow them in MultiplicityRange (see
+        // SysML 8.2.2.6.6 and KerML 8.2.5.11).
+        List<OperatorExpression> operatorExpressions = EMFUtils.allContainedObjectOfType(root, MultiplicityRange.class)
+                .flatMap(multiplicityRange -> multiplicityRange.getOwnedMember().stream())
+                .filter(OperatorExpression.class::isInstance)
+                .map(OperatorExpression.class::cast)
+                .filter(operatorExpression -> Objects.equals(operatorExpression.getOperator(), ".."))
+                .toList();
+        for (OperatorExpression operatorExpression : operatorExpressions) {
+            Element owner = operatorExpression.getOwner();
+            if (owner != null) {
+                for (Feature parameter : operatorExpression.getParameter()) {
+                    Expression parameterValue = parameter.getValuation().getValue();
+                    // Only LiteralExpressions and FeatureReferenceExpressions can be used in a MultiplicityRange
+                    if (parameterValue instanceof LiteralExpression || parameterValue instanceof FeatureReferenceExpression) {
+                        OwningMembership newOwningMembership = SysmlFactory.eINSTANCE.createOwningMembership();
+                        owner.getOwnedRelationship().add(newOwningMembership);
+                        newOwningMembership.getOwnedRelatedElement().add(parameterValue);
+                    } else if (parameterValue instanceof FeatureChainExpression featureChainExpression) {
```

Comment:
See comment in the test but I think this case does not happen.

Reply:
I removed this part